google-cloud-core/pom.xml

```diff
@@ -116,6 +116,11 @@
       <version>3.0.0</version>
     </dependency>
     <dependency>
+      <groupId>com.google.protobuf</groupId>
+      <artifactId>protobuf-java-util</artifactId>
+      <version>3.0.0</version>
```

Comment:
I noticed that protobuf-java-util has a dependency on Guava. It doesn't expose Guava on its surface, but at the same time, it there isn't a shaded variant of it either. We are currently working towards providing a fully shaded variant of google-cloud-java soon (alongside the current variant where Guava isn't shaded).